processes/oclcClassify.py

```diff
@@ -106,19 +107,38 @@ def frbrizeRecord(self, record):
 
             # Check if this identifier has been queried in the past 24 hours
             # Skip if it has already been looked up
-            if identifier and self.checkSetRedis(
-                'classify', identifier, idenType
-            ):
+            if identifier and self.checkSetRedis('classify', identifier, idenType):
                 continue
 
             try:
-                self.classifyRecordByMetadata(
-                    identifier, idenType, author, record.title
-                )
+                # TODO: switch this to classify_record_by_metadata_v2
+                self.classifyRecordByMetadata(identifier, idenType, author, record.title)
             except ClassifyError as err:
                 logger.warning('Unable to Classify {}'.format(record))
                 logger.debug(err.message)
 
+    def classify_record_by_metadata_v2(self, idenfitier, id_type, author, title):
+        # TODO: deprecate classify manager
+        classify_manager = ClassifyManager(iden=idenfitier, idenType=id_type, author=author, title=title)
+        catalog_manager = OCLCCatalogManager()
+
+        # TODO: move generate_search_query to utils file
+        search_query = classify_manager.generate_search_query()
+
+        # TODO: return brief records directly 
+        # TODO: check to see we don't return the record's bib
+        related_oclc_bibs = catalog_manager.query_brief_bibs(search_query)
+
+        for related_oclc_bib in related_oclc_bibs.get('briefRecords', []):
+            # TODO: returns only 10 records, should we get all of the records?
+            # TODO: not sure if actually have to send this to the oclc catalog process
+            related_oclc_numbers = catalog_manager.get_related_oclc_numbers(related_oclc_bib['oclcNumber'])
```

Comment:
I feel like we should, but we can test to see what happens if we don't vs. if we do send it to the oclc catalog process.

Reply:
Agreed - I think we should send them over and then make a change later on if we decide we can simplify.